main.py

```diff
@@ -3,14 +3,34 @@
 
 #read in the training files
 clinical = pd.read_csv('data/raw/train_cli.tsv', sep='\t')
-protein = pd.read_csv('data/raw/train_pro.tsv', sep='\t')
+protein = pd.read_csv('data/raw/train_pro.tsv', sep='\t').T
+labels = pd.read_csv('data/tidy/sum_tab_1.csv', sep=',')
+
+#create the labels for which samples ahve been mislabeled
+mismatch_labels = labels.mismatch.tolist()
+
+#make the row names of clinical data equal to the first column's content
+clinical.index = clinical['sample'].tolist()
+
+#remove the first column
+clinical = clinical.iloc[:,1:]
+
+#create column headers for protein data
+protein.columns = protein.iloc[0]
+protein = protein.iloc[1:]
 
 #impute missing values with 0
 protein = protein.fillna(0)
 clinical = clinical.fillna(0)
 
-#remove the sample labels and transform the data such that proteins are features
-protein_data = protein.iloc[:,1:].T
+#combind teh clinical and the protein data

```

Comment:
Here and in other places in the file, I think "combind" should be "combined"?

And "teh" -> "the" 🙂